content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/android.md

```diff
@@ -179,26 +179,76 @@ In addition to [tracking resources automatically][6], you can also track specifi
+    sdkCore.clearAllData()
+}
+```
+
+### Stop data collection
+
+You can use the `StopInstance` API to stop the SDK instance assigned to the given name (or the default instance if the name is null) from collecting and uploading data further.
+
+```kotlin
+   fun stopInstance(instanceName: String? = null) {
+       synchronized(registry) {
+           val instance = registry.unregister(instanceName)
+           (instance as? DatadogCore)?.stop()
+       }
+   }
+```
 
-In addition to the [default RUM attributes][3] captured by the RUM Android SDK automatically, you can choose to add additional contextual information, such as custom attributes, to your RUM events to enrich your observability within Datadog. Custom attributes allow you to slice and dice information about observed user behavior (such as cart value, merchant tier, or ad campaign) with code-level information (such as backend services, session timeline, error logs, and network health).
+### Control event buildup
 
-### Track User Sessions
+Many operations, such as data processing and event input/output, are queued in background threads. To handle edge cases where the queue has grown so much that there could be delayed processing, high memory usage, or Application Not Responding (ANR) errors.
 
-Adding user information to your RUM sessions makes it easy to:
-* Follow the journey of a given user
-* Know which users are the most impacted by errors
-* Monitor performance for your most important users
+You can control the buildup of events on the SDK with the `setBackpressureStrategy` API. This API ignores new tasks if a queue reaches 1024 items.
 
-{{< img src="real_user_monitoring/browser/advanced_configuration/user-api.png" alt="User API in RUM UI" >}}
+```kotlin
+   fun setBackpressureStrategy(backpressureStrategy: BackPressureStrategy): Builder {
+       coreConfig = coreConfig.copy(backpressureStrategy = backpressureStrategy)
+       return this
+   }
+```
+
+See an [example of this API][12] being used.
+
+### Set remote log threshold
+
+You can define the minimum log level (priority) to send events to Datadog in a logger instance. If the log priority is below the one you set at this threshold, it does not get sent. The default value is -1 (allow all).
+
+```kotlin
+   fun setRemoteLogThreshold(minLogThreshold: Int): Builder {
+       minDatadogLogsPriority = minLogThreshold
+       return this
+   }
+```
 
-The following attributes are **optional**, you should provide **at least one** of them:
+## Track custom global attributes
 
-| Attribute  | Type | Description                                                                                              |
+In addition to the [default RUM attributes][3] captured by the RUM Android SDK automatically, you can choose to add additional contextual information, such as custom attributes, to your RUM events to enrich your observability within Da                                                       |
```

Comment:
I'm not sure if this is intentional, but this is rendering as a table header cell in the docs. Also, it looks like the last sentence might've gotten cut off here.

Reply:
Yeah something got messed up here, thanks for the catch!